config/services.yaml

```
@@ -78,6 +82,12 @@ services:
     Bolt\Event\Listener\UserAvatarLoadListener:
         tags:
             - { name: doctrine.event_listener, event: postLoad }
+    
+    Bolt\Event\Listener\TranslatableListener:
+        tags:
```

```suggestion
    Bolt\Event\Listener\TranslatableListener:
        arguments:
            $translatableFetchMode: '%bolt.doctrine_behaviors_translatable_fetch_mode%'
            $translationFetchMode: '%bolt.doctrine_behaviors_translation_fetch_mode%'
        tags:
```

Once we have upgraded to Symfony 6 we can probably use the `#[Autowire]` on the class instead, but for now we should do it like this.